opus/application/apps/ui/views.py

```diff
@@ -80,22 +81,23 @@ def get_context_data(self, **kwargs):
         return context
 
 @never_cache
-def api_last_blog_update(request):
-    """Return the date of the last blog update.
+def api_notifications(request):
+    """Return the HTML for any pending notifications and the date of the last blog update.
```

Comment:
Comment exceeds 80 character limit - shorten the first line to fit within 80 character, and fix rest of comment to fit as well.